.github/workflows/docker-image.yml

```
@@ -0,0 +1,31 @@
+name: deploy-docker
+
+on:
+  push:
+    branches: [ development ]
+  pull_request:
+    branches: [ development ]
+
+jobs:
+  build:
+
+    timeout-minutes: 10
```

```suggestion
    timeout-minutes: 60
```
The last run was cancelled after 10m 9s, I think it needs more time. If it works with 60m we can reduce it later.